### PR TITLE
Adds annotation type component interface to development fixtures

### PR DESCRIPTION
**Pull request by miriam-groeneveld**
Needed to be able to test https://github.com/DIAGNijmegen/rse-gcapi/pull/144

### CODE REVIEW

**Comment by chrisvanrun** [comment]
This doesn't work because the `strict=True` is a python 3.10 feature.

File: scripts/development_fixtures.py [hunk 1]
```diff
@@ -535,6 +535,14 @@ def _create_reader_studies(users):
...
     )
     image.save()
+    annotation_interface = ComponentInterface(
+        store_in_database=True,
+        relative_path="annotation.json",
+        slug="annotation",
+        title="Annotation",
+        kind=ComponentInterface.Kind.TWO_D_BOUNDING_BOX,
+    )
+    annotation_interface.save()
```
Comment (chrisvanrun) [inline scripts/development_fixtures.py:545]: This should likely be cherry-picked on top of:
- https://github.com/comic/grand-challenge.org/pull/2715

**Review by chrisvanrun — CHANGES_REQUESTED** [review]
Doesn't work for python 3.8 (default python Django runs underatm)

**Review by jmsmkn — APPROVED** [review]
LGTM, there is a tight coupling from the gcapi tests to some of the things in this file, probably they should be separated out and added to a new separate set of fixtures managed in gcapi itself.

**Pull request merged**